packages/client/README.md

```diff
@@ -0,0 +1,69 @@
+# `@lens-protocol/client`
+
+This package contains low level client for GraphQL Lens API.
+
+**Its interface will change without notice, use it at your own risk.**
```

Comment:
I think this paragraph comes from the README file of an internal package.

Given this is a publica facing package something akin to what we wrote in `packages/react/README.md` is more appropriate:

```suggestion
> **Warning**
>
> The Lens SDK is still in its initial development phase. Anything MAY change at any time.
> This is a Developer Preview aimed primarily at existing integrators so to gather [early feedback](https://github.com/lens-protocol/lens-sdk/discussions/48).
```